api/.pipeline/lib/api.deploy.js

```diff
@@ -37,6 +37,7 @@ const apiDeploy = async (settings) => {
         // Persistent Volume
         VOLUME_CAPACITY: phases[phase].volumeCapacity,
         // BioHub Platform (aka: Backbone)
+        BACKBONE_PUBLIC_API_HOST: phases[phase].backbonePublicApiHost,
```

Comment:
Hmm, it seems like this is never actually used in the API anywhere. Maybe we should just remove it here, and remove it from the env-config schema

Reply:
Its passed into the compose.yml too, but I guess is never referenced after that

Reply:
Yea doesn't look like it hey. Dropping.